modules/options/src/main/scala/scala/build/Artifacts.scala

```diff
@@ -654,6 +653,17 @@ object Artifacts {
     dependencies: Seq[Positioned[coursier.Dependency]],
     resolutionError: coursier.error.ResolutionError.Simple
   ) = resolutionError match {
+    // FIXME This if may not be enough when new toolkits are added
```

Comment:
maybe add some context on what needs to be done here? potentially link to a new ticket?

Reply:
Actually I think my comment is not valid since the names of the artifacts for toolkits should always the same, no matter the library author. Am I right?

Reply:
yeah, I think so.
Then it seems the `// FIXME` comment is invalid.